### DISCUSSION TITLE
Anonymous access to get files but not list them?

### DISCUSSION THREAD

**Discussion by ser** [minio/minio#18334] [Q&A]
i am trying to make a bucket to allow anonymous download but not listing.

i found somewhere that you could before:

```mc policy get-json minio/bucket >> bucket.json```

edit that file removing `ListBucket` and upload back using

and upload with ```mc policy set-json minio/bucket bucket.json```

- but new versions of minio seem to don't have `mc policy`?

Any help appreciated.

**Comment by harshavardhana** [comment]
Its called `mc anonymous` now

**Comment by ser** [comment]
No it's not a resolved question. `mc anonymous` does not give possibility to remove listing.

```
mc anonymous get-json minio/bucket | jq > bucket.json
```
editing file, removing  `s3:ListBucket`...
```
mc anonymous set-json minio/bucket bucket.json
Access permission for `minio/bucket` is `download`
```
getting it back...
```
mc anonymous get-json minio/bucket | jq > bucket.json
```
and find out nothing has changed,  `s3:ListBucket` is still in there.

Any help really appreciated.

**Reply by harshavardhana** [reply]
You need to remove it manually, not sure what else you are asking.

**Reply by ser** [reply]
what do you mean `manually`? i am asking how to do it, it can be `manually` if required :)

**Comment by danielcommesse** [comment]
The solution in this discussion:

https://github.com/minio/minio/discussions/18451#discussioncomment-10959112

Hope it helps!
